ospfd/ospf_packet.c

```diff
@@ -2427,6 +2430,11 @@ static int ospf_check_network_mask(struct ospf_interface *oi,
 	    || oi->type == OSPF_IFTYPE_VIRTUALLINK)
 		return 1;
 
+	/* Ignore mask check for max prefix length (32) */
```

Comment:
@pguibert6WIND this change is added to ignore network mask check for interface is configured with /32 prefix